go/oasis-test-runner/oasis/client.go

```diff
@@ -63,6 +66,10 @@ func (client *Client) ModifyConfig() error {
 		client.Config.Runtime.Provisioner = client.runtimeProvisioner
 	}
 
+	if client.batchSize > 0 {
```

Comment:
To avoid this you can always allow 0 and use a default value if set to 0. Or be strict as you are and not allow 0 at all. Both options are fine.